.travis.yml

```diff
@@ -17,85 +12,76 @@ before_install:
 install:
   -
 
-env:
-  -JDK=Java8
-  -JDK=Java9
-  -JDK=Java10
-  -JDK=Java11-EA
-  -JDK=Java12-EA
-
```

Comment:
Typically, declaring an environment up front like this is used for an automatic build matrix, where builds run for all n environments for all m jdks. Anyway, the environment can still be declared lazily it seems, for the Java 11 and 12 EA builds.